README.md

```diff
@@ -1,10 +1,18 @@
-# Docker images for neuroscience
+# Biological neuronal network simulations with Python
```

Comment:
We also provide images for neuroscience data analysis (Neo, Elephant, etc.), so I'd like to leave the title of the top-level README unchanged. All the rest of the changes are good.